src/custom/pages/Account/Balances.tsx

```diff
@@ -55,6 +56,8 @@ export default function Profile() {
   const setSwapVCowStatus = useSetSwapVCowStatus()
   const swapVCowStatus = useSwapVCowStatus()
 
+  const isMetaMask = (connector as MetaMask)?.provider?.isMetaMask
```

Comment:
That is what I thought, but its not what it looks like.

`export function getIsMetaMask(): boolean {
  return window.ethereum?.isMetaMask ?? false
}`

It actually just checks if there is a Metamask wallet and not if the current connected wallet is metamask.
I think I've made some mistakes before where I've used that method instead of this so I will have to check that again.